editor/plugins/item_list_editor_plugin.cpp

```diff
@@ -391,7 +467,7 @@ ItemListEditorPlugin::ItemListEditorPlugin(EditorNode *p_node) {
 	item_list_editor->hide();
 	item_list_editor->add_plugin(memnew(ItemListOptionButtonPlugin));
 	item_list_editor->add_plugin(memnew(ItemListPopupMenuPlugin));
-	item_list_editor->add_plugin(memnew(ItemListItemListPlugin));
+	item_list_editor->add_plugin(memnew(ItemListItemListPlugin));	
```

Comment:
Accidental change? Just inserted a tab. I would suggest reviewing your changes yourself to ensure as little of these sneak through.